.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
**Search terms you've used**

<!-- What search terms have you used to check whether this feature has been requested before? -->